content/learning-paths/cross-platform/integer-vs-floats/implicit-conversions.md

```diff
@@ -114,15 +162,15 @@ golden_ratio:
         ret
 ```
 
-You now see that the instruction `scvtf` is executed twice, which converts from integer to floating point, so that the quantities can be divided using floating point with the instruction `fdiv`. The `scvtf` instruction has up to 6 CPU cycles latency in Neoverse N1 and 3 cycles in Neoverse N2. Imagine a complicated expression that has lots of similar implicit conversions, executed millions or billions of times, it would definitely underperform.
+You now see that `scvtf` instruction is executed twice, which converts from integer to floating point, so that the quantities can be divided using floating point with the `fdiv` instruction. The `scvtf` instruction has up to 6 CPU cycles latency on the Neoverse N1 processor and 3 cycles on the Neoverse N2. Imagine a complicated expression many similar implicit conversions, executed millions or billions of times, it would definitely under perform.
```

Comment:
underperform is actually the right word for this.

Reply:
@markos Thanks, I will correct!